src/scripts/game.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+// import:
+const {
+  mobileSwipes,
+} = require('./mobile');
+
+const {
+  gameScore,
+  gameField,
+  allCells,
+  highestScore,
+  startBtn,
+  messageBeforeStart,
+  messageDuringTheGame,
+  messageAfterLose,
+  messageAfterWin,
+} = require('./variables');
+
+const {
+  arrowsPlayHandler,
+  createItemInRandomEmptyField,
+} = require('./move');
+
+// main function:
+function gameInit() {
+  highestScore.innerText = localStorage.getItem('score') || 0;
+
+  startBtn.addEventListener('click', (e) => {
+    if (startBtn.innerText === 'Start') {
+      startBtn.innerText = 'Restart';
+      startBtn.classList.remove(`start`);
+      startBtn.classList.add(`restart`);
+
+      messageDuringTheGame.classList.remove('hidden');
+      messageBeforeStart.classList.add('hidden');
+
+      createItemInRandomEmptyField();
+      createItemInRandomEmptyField();
+
+      mobileSwipes(gameField);
+      document.body.addEventListener('keyup', arrowsPlayHandler);
```

Comment:
I think this logic can be moved to separate method
I mean, I understand that it can't be reused, because it's a specific logic for one case, but this code inside if / else block is hard to read

It's not a critical issue, but would be nice to do it